src/compiler-debugging.md

```diff
@@ -149,10 +149,10 @@ $ # Cool, now I have a backtrace for the error
 These crates are used in compiler for logging:
 
 * [log]
-* [env-logger]: check the link to see the full `RUSTC_LOG` syntax
+* [env-logger]
 
 [log]: https://docs.rs/log/0.4.6/log/index.html
-[env-logger]: https://docs.rs/env_logger/0.4.3/env_logger/
+[env-logger]: https://docs.rs/env_logger/0.8.1/env_logger/
```

Comment:
Let's use docs.rs to get the latest version so we don't have to keep updating it.
```suggestion
[env-logger]: https://docs.rs/env_logger
```